content/docs/authorization/Migration guide from v1 to v2/_index.md

```diff
@@ -0,0 +1,179 @@
+  - Volumes belonging to a tenant are identified using **Volume Prefix** configured in tenant CR.
+  - Volumes without **Volume Prefix** will not be accounted in usage capacity calculation as ownership of volume is unknown without the volume prefix.
+  - User should rename all volumes that are needed to be accounted with **Volume Prefix** before migration to v2.
+
+## Storage Systems
+
+In CSM for Authorization v1 setup, list the storage to get all the storages configured in the environment.
+Example: :
+
+```
+karavictl storage list --admin-token admintoken.yaml --addr csm-authorization.host.com
+
+{
+  "storage": {
+    "powerflex": {
+      "3000000000011111": {
+        "Endpoint": "https://1.1.1.1",
+        "Insecure": true,
+        "Password": "(omitted)",
+        "User": "admin"
+      }
+    }
+  }
+}
+```
+In CSM for Authorization v2 Storage are created using CRDs. For each Storage in a v1 environment, create using the CRD, example:
+
+```
+kubectl create -f controller/config/samples/csm-authorization_v1_storage.yaml
+```
+Content of csm-authorization_v1_storage.yaml
+```yaml
+apiVersion: csm-authorization.storage.dell.com/v1
+kind: Storage
+metadata:
+name: powerflex
+spec:
+# Type of the storage system. Example: powerflex, powermax, powerscale
+type: powerflex
+endpoint: https://1.1.1.1
+# System ID of the backend storage array
+systemID: 3000000000011111
+# Vault is the credential manager for storage arrays
+vault:
+identifier: vault0
+kvEngine: secret
+path: csm-authorization/powerflex/1a99aa999999aa9a
+# SkipCertificateValidation is the flag to skip certificate validation
+skipCertificateValidation: true
+# PollInterval is the polling frequency to test the storage connectivity
+pollInterval: 30s
+```
+
+
+## Role and Role Binding
+
+In CSM for Authorization v2, role creation is simpler. User will not be required to bind the role, only thing user needs to do is create roles that they needed.
+
+List all the roles that are created in CSM for Authorization v1 setup.
+Example:
+```
+karavictl role list --admin-token admintoken.yaml --addr csm-authorization.host.com
+```
+```
+{
+  "CSIGold": [
+    {
+      "storage_system_id": "3000000000011111",
+      "pool_quotas": [
+        {
+          "pool": "mypool",
+          "quota": 32000000
+        }
+      ]
+    }
+  ],
+  "CSISilver": [
+    {
+      "storage_system_id": "3000000000011111",
+      "pool_quotas": [
+        {
+          "pool": "mypool",
+          "quota": 16000000
+        }
+      ]
+    }
+  ]
+}
+```
+For each role in v1 setup, create roles using csmrole CRD.
+Example
+```
+kubectl create -f controller/config/samples/csm-authorization_v1_csmrole.yaml
+```
+csm-authorization_v1_csmrole.yaml file for **CSIGold** role in v1 setup will look like following example file:
+```yaml
+apiVersion: csm-authorization.storage.dell.com/v1
+kind: CSMRole
+metadata:
+labels:
```

Comment:
can we remove the labels?

Reply:
Its there in sample files, I deleted other commented info which are added for in-line help to user, and leave the rest as it is in sample file